grpcurl/tests/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+TESTDIR="$(dirname "${0}")"
+PLANDIR="$(dirname "${TESTDIR}")"
+SKIPBUILD=${SKIPBUILD:-0}
+
+source "${PLANDIR}/plan.sh"
+
+if [ "${SKIPBUILD}" -eq 0 ]; then
+  set -e
+  pushd "${PLANDIR}" > /dev/null
+  build
+  popd > /dev/null
+  set +e
+fi
+
+source "${PLANDIR}/results/last_build.env"
+hab pkg install --binlink --force "${PLANDIR}/results/${pkg_artifact}"
+
+# if the version output (sent to stderr) is not what we expect, this will exit 1
+grpcurl -version 2>&1 >/dev/null | grep -q "build v${pkg_version}"
```

Comment:
This should ideally use `hab pkg exec`
Some reasons why are documented [here](https://github.com/habitat-sh/core-plans/blob/master/docs/dev/policy_documents/standards-for-tests.md#examples-of-why-hab-pkg-exec-should-be-used)